### PR TITLE
Deflake testPostPolicy by means of repeated tapping

### PR DESCRIPTION
**Pull request by cobbal**
[run-nightly]

(sorry for the unfair blame, narmstrong)

refs: none
affects: none
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: None

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.52% | 93.52% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by cobbal** [comment]
bitrise-github communications seem flaky sometimes... this is the new nightly build https://app.bitrise.io/build/4efdcf62dd0dd536

**Comment by armstrongnate** [comment]
Uh oh that nightly build failed too

File: TestsFoundation/TestsFoundation/UITests/CoreUITestCase.swift [hunk 1]
```diff
@@ -448,6 +448,19 @@ open class CoreUITestCase: XCTestCase {
     }
 
     open func doGraphQLMock(request: URLRequest) -> Data {
+        #if false
```
Comment (cobbal) [inline TestsFoundation/TestsFoundation/UITests/CoreUITestCase.swift]: set this to true for easy copy-pasting into graphiql

**Pull request merged**